### DISCUSSION TITLE
How to unset a color?

### DISCUSSION THREAD

**Discussion by henningBunk** [JLLeitschuh/ktlint-gradle#464] [Q&A] [answered]
Hi there!
When I run ktlint and let it create a plain report the textfile is littered with some kind of encoding errors:

eg: `[90m/Users/henning.bunk/repos/work/wetter-app/[0mbuild.gradle.kts[90m:[0m38[90m:9:[0m Missing space after //`

I got the tip that they are very likely color codes.

I did not use the option `outputColorName.set("WHITE")` though. Using it changes nothing.   
Trying to null the color with `outputColorName.set(null)` does not compile.

Is there a default color which is set? Can this be unset?

Thanks!

**Comment by Tapchicoma** [comment, marked as answer]
You should set `coloredOutput.set(false)` to remove color codes from plain report.
